generator/generate-library-list.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+#cmd line:: ./generate-library-list.sh -c 9ff8aa652e195965037706a3455a8e8aa318be5c
+
+while getopts c: flag
+do
+    case "${flag}" in
+        c) commitish=${OPTARG};;
+    esac
+done
+echo "monorepo commitish to checkout: $commitish";
+
+
+# install jq for json parsing if not already installed
+sudo apt-get -y install jq
+
+# download the monorepo, need to loop through metadata there
+git clone https://github.com/googleapis/google-cloud-java.git
+
+# switch to the specified release commitish
+cd ./google-cloud-java
+if [ -z "$commitish" ];
+  then echo "No commitish provided, using HEAD.";
+  else git checkout $commitish;
+fi
+
+cd -
+
+# start file, always override is present
+filename=./library_list.txt
+echo "# api_shortname, googleapis-folder, distribution_name:version" > $filename
+
+count=0
+for d in ./google-cloud-java/*java-*/; do
+  # parse variables from .repo-metadata.json
+  language=$(cat $d/.repo-metadata.json | jq -r .language)
+  api_shortname=$(cat $d/.repo-metadata.json | jq -r .api_shortname)
+  distribution_name=$(cat $d/.repo-metadata.json | jq -r .distribution_name)
+  library_type=$(cat $d/.repo-metadata.json | jq -r .library_type)
+  transport=$(cat $d/.repo-metadata.json | jq -r .transport)
```

Comment:
The transport attribute in `repo-metadata.json` is not the source of truth, the source of truth is the bazel config in googleapis, like [here](https://github.com/googleapis/googleapis/blob/1051d7d7ffe66fdbdc14cbd2ec2eed3b8741cb4a/google/cloud/asset/v1/BUILD.bazel#L80), we've seen a lot of discrepancies before. If all we want to do is excluding rest only clients, then we can include everything in the mono repo except for `compute`